### ISSUE TITLE
[Frontend] Refactoring Frontend

### ISSUE THREAD

**Issue by djeck1432** [djeck1432/spotnet#396]
🚨 Attention: Please read the guidelines before applying; otherwise, you will be ignored. 🚫📋
0. Please, join the telegram group to keep in touch https://t.me/+m0IGRiBvLtsyMzIy

1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. In your application should be telegram handler to contact you and ETA ( ETA > 24 hours won't be considered at all)
3.  Renamed `log out` button to `Disconnect`
<img width="275" alt="Screenshot 2024-12-17 at 17 15 30" src="https://github.com/user-attachments/assets/30ed2f14-44da-478c-b60c-7bb18c404180" />

4. Use for error message `react-toastify` in dashboard page or any other places
5.  Error message should disappear  after 3 seconds
6. Add successful deposit notification if [`sendTransaction`](https://github.com/djeck1432/spotnet/blob/main/frontend/src/services/transaction.js#L8C23-L8C38) with transaction id in it
7. Hide transaction id in text `transaction id`
<img width="304" alt="Screenshot 2024-12-17 at 17 40 47" src="https://github.com/user-attachments/assets/beb5504b-6a8f-451a-a115-3cc99582a92a" />

8. In dashboard page, rename `Balance` to `Position balance`

**Comment by nottherealalanturing** [comment]
can i tackle this

**Comment by 1nonlypiece** [comment]
Can i work on this, I like strawberry. TG: kazamada

**Comment by fabrobles92** [comment]
Hello I am Fab, a member of dojo coding and also a full stack engineer with experience in Next Js, Vue and plain react, also I have plenty of contribution to web3 projects where that experience can help me deliver a quality solution.

I estimate to make a PR today.

**Comment by omsant02** [comment]
I would like to work on this issue

I like strawberry.
My telegram handle: omsanto02

My approach is to first rename the logout button to disconnect 
next i would implement the react-toasts for modal for error display,
implement notification for successful deposit transaction, and lastly
will rename the balance to position balance

ETA : 1 day

**Comment by Emmanex01** [comment]
My approach on this, 
Firstly, I review the code guidelines as instructed above. 
Secondly, implement the required changes in all components mentioned above. 
Thirdly, Ensure that the code flows the guidelines as stated. 
TG: @Emmanuel_too 
I like strawberry 🍓 
ETA: 24hrs

**Comment by BigBen-7** [comment]
I like strawberry 
My approach on this,
Would be to review the code guidelines as instructed 
Then, implement the required changes in all components as stated.
Lastly, I would make sure  that the code flows as the guidelines  stated.
Telegram: @BigBen1909
ETA: 24hrs

**Comment by GoSTEAN** [comment]
Can I work on this? 
I love strawberry

**Comment by KeneePatel** [comment]
Hi there! I've already worked on several projects here on ODHack, notably kakarot and starknet.js, where I successfully completed multiple issues. Of course, I'll make sure to follow all the community rules and contributing guidelines carefully, as I've done before. I have good experience working on React, therefore I will make sure all the changes are done properly.

I like strawberry.

ETA: 8h
TG Handle: @patelkenee
